spinnaker/deck
# feat(provider/azure): Add healthSettings to server groups

Azure supports health checks on server groups, and clouddriver
already supports sending healthSettings via the pipeline JSON.
This change adds this functionality to the Azure server group
configuration on the UI.

**Screenshots**:
HTTP:

![image](https://user-images.githubusercontent.com/910168/131192113-3d048c07-c33a-4271-b65b-2061f6cb1331.png)

TCP:

![image](https://user-images.githubusercontent.com/910168/131192132-ba49eb27-2f9b-43ce-b5ea-d115f4ee1810.png)

N/A (null):

![image](https://user-images.githubusercontent.com/910168/131192153-fd8c496f-bf17-429a-95d4-ef9005dcf69a.png)


**Testing done**:

1. Add HTTP protocol health check to server group and deploy
2. Add TCP protocol health check to server group and deploy
3. Remove health check from server group and deploy
4. Clone server group using HTTP protocol health check
5. Clone server group using TCP protocol health check
6. Clone server group with N/A (null) health check
